calico_containers/calico_ctl/bgp.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2015 Metaswitch Networks
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Usage:
+  calicoctl bgp peer add <PEER_IP> as <AS_NUM>
+  calicoctl bgp peer remove <PEER_IP>
+  calicoctl bgp peer show [--ipv4 | --ipv6]
+  calicoctl bgp node-mesh [on|off]
+  calicoctl bgp default-node-as [--as=<AS_NUM>]
+
+
+Description:
+  Configure default global BGP settings for all nodes. Note: per-node settings
+  will override these globals for that node.
+
+Options:
+ --ipv4    Show IPv4 information only.
+ --ipv6    Show IPv6 information only.
+"""
+import sys
+from utils import client
+from pycalico.datastore_datatypes import BGPPeer
+from netaddr import IPAddress
+from utils import check_ip_version
+from prettytable import PrettyTable
+from utils import get_container_ipv_from_arguments
+
+
+def bgp(arguments):
```

Comment:
Each calico_ctl/* file has a new function which shares the same name as the file. This handles the function dispatching within the submodule.